server/sql_app/main.py

```diff
@@ -43,9 +63,71 @@ def get_db():
 # operation function and use that session.
 
 
+# To make sure Exceptions aren't silently ignored:
+# https://stackoverflow.com/questions/27297638/when-asyncio-task-gets-stored-after-creation-exceptions-from-task-get-muted/27299160#27299160
+def handle_result(fut):
+    if fut.exception():
+        fut.result()  # This will raise the exception.
```

Comment:
Should we also be removing the task from the tasks list?  If not, will we ever run out of available memory because we have all the tasks run since process startup in `TASKS`?